pulpcore/app/response.py

```diff
@@ -23,5 +25,6 @@ def __init__(self, result, request):
                 the response.
             request (rest_framework.request.Request): Request used to generate the _href urls
         """
-        task = {"task": reverse('tasks-detail', args=[result.id], request=None)}
-        super().__init__(data=task, status=202)
+        task = Task.objects.get(job_id=result.id)
+        resp = {"task": reverse('tasks-detail', args=[task.id], request=None)}
```

Comment:
Same issue with 'id'

Reply:
This one is ours, so this could become `pk` possible. I don't have a preference.

Reply:
I think this really should be pk then given the redis 'id' to ease confusion where we are able.